cloud_connectors/azure/mqtt_connector/azure_function/src/run.cs

```diff
@@ -19,20 +20,30 @@ namespace Microsoft.ESDV.CloudConnector.Azure {
     /// <summary>
     /// This class contains the info to target an Azure Digital Twin instance.
     /// </summary>
-    public class DigitalTwinsInstance {
-        public string model_id { get; set; }
-        public string instance_id { get; set; }
-        public string instance_property_path { get; set; }
-        public string data { get; set; }
+    public class DigitalTwinsInstance
+    {
+        [JsonPropertyName("model_id")]
+        public string ModelId { get; set; }
+
+        [JsonPropertyName("instance_id")]
+        public string InstanceId { get; set; }
+
+        [JsonPropertyName("instance_property_path")]
+        public string InstancePropertyPath { get; set; }
+
+        [JsonPropertyName("data")]
+        public string Data { get; set; }
     }
 
-    public class MQTTConnectorAzureFunction {
+    public class MQTTConnectorAzureFunction
```

Comment:
Please add a class method.

Reply:
I'm not sure what you're asking here

Reply:
Sorry, meant "Please add a class comment".